charts/victoria-metrics-anomaly/templates/controller.yaml

```diff
@@ -50,8 +57,15 @@ spec:
             {{ toYaml . | nindent 12 }}
             {{- end }}
           {{- end }}
-          {{- with .Values.env }}
-          env: {{ toYaml . | nindent 10 }}
+          {{- if and .Values.env .Values.persistentVolume.enabled }}
```

Comment:
`.Values.env` will be omitted if `.Values.persistentVolume.enabled=false`

Reply:
Good catch, thanks! 
Fixed in latest commit